### DISCUSSION TITLE
✔ OSHP logo and mantra

### DISCUSSION THREAD

**Discussion by righettod** [oshp/oshp-tracking#13] [Ideas]
Hi,

I was thinking that it can be fun to have a **proper logo** and **its mantra** for the OSHP project reflecting the idea and mindset of the project.

What do you think?

**Comment by riramar** [comment]
That's a great idea! But I'm really bad in design. 😭 
Do you have anything in mind?

**Comment by righettod** [comment]
Nothing special in mind, I will work on some proposal...

Feel free to propose anything you have in mind for the mantra 😃

**Comment by righettod** [comment]
💬 Below is a collection of proposal for the mantra:

- When exploitation become harder.
- Became unexploited.
- Reach non exploitability.
- Welcome to exploitation's hell.
- Defense in depth is our credo.
- Defense in depth as credo.
- From browser to cage.

🖼 Below is a collection of proposal for the logo ([source](https://editor.freelogodesign.org/en/logo/edit/5ca91a3a6b0a423bae94cfb840133acf?companyName=OSHP&view=icons)):

![image](https://user-images.githubusercontent.com/1573775/195974685-ab958042-cbc2-4473-891f-5d305ea7abfb.png)

![image](https://user-images.githubusercontent.com/1573775/195974721-6dede3c7-f284-4b17-a7cb-82d06ed8a346.png)

![image](https://user-images.githubusercontent.com/1573775/195974735-a23245ce-5bdc-4e39-883b-8e4cd234fa0c.png)

![image](https://user-images.githubusercontent.com/1573775/195974834-7fe2964b-3b28-4eff-b86e-dec3e068c908.png)

![image](https://user-images.githubusercontent.com/1573775/195974750-683d9fbf-ccab-4450-9f11-f4e1137558f5.png)

**Comment by righettod** [comment]
I have a preference for the mantra `Defense in depth as credo.` because it represent well the objective of a security header.

I have a preference for this logo for the same reason than for the mantra and make it me think to the `defense in depth` principle:

![image](https://user-images.githubusercontent.com/1573775/195974750-683d9fbf-ccab-4450-9f11-f4e1137558f5.png)

**Comment by riramar** [comment]
I'm fine with your proposal. I also liked the one below.

![image](https://user-images.githubusercontent.com/6748242/195995511-e18553d5-3095-48a4-8c20-945911bf1ce6.png)

**Reply by righettod** [reply]
It is true that the one you mentioned is cool too and easier to draw 👍

**Comment by righettod** [comment]
✅ OK, so let's go  for the following content:

**Image:**

![195995511-e18553d5-3095-48a4-8c20-945911bf1ce6](https://user-images.githubusercontent.com/1573775/201485112-afd5266d-b9e1-4568-b383-c44f4b79df10.png)

**Mantra:**

`Defense in depth as a credo`

**Comment by righettod** [comment]
🚧 I started working with a designer, via the [Fiverr](https://fiverr.com) platform, to create the logo.

**Comment by righettod** [comment]
[PR](https://github.com/OWASP/www-project-secure-headers/pull/104) submitted 💯

**Comment by righettod** [comment]
[PR](https://github.com/OWASP/owasp-swag/pull/51) submitted to the SWAG repo 😃
